tests/countries/test_nauru.py

```
@@ -0,0 +1,275 @@
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.nauru import Nauru, NR, NRU
+from tests.common import CommonCountryTests
+
+
+class TestNauru(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Pylint (3.3.7)

[warning] Number of parameters was 4 in 'TestCase.setUpClass' and is now 1 in overriding 'TestNauru.setUpClass' method

(W0221)